Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -1,25 +1,59 @@
-﻿namespace BootCamp.Chapter1
+﻿using System;
+namespace BootCamp.Chapter1
 {
     public static class ArrayOperations
     {
+
+        public static bool ValidateArrayForManipulation(int[] array)
+        {
+            if (array == null)
+                return false;
+            if (array.Length == 0)
+                return false;
+            else
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'else' block.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4887772)